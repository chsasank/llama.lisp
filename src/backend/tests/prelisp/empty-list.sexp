(c-lisp
    (define ((main void))
        (declare arr1 (ptr int))
        ,@(arr-init arr (1 2 3) 5)

        (declare arr2 (ptr int))
        ,@(arr-init arr2 () 5)))
