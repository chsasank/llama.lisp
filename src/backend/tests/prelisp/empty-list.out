(c-lisp
  (define ((main void))
    (declare arr1 (ptr int))
    (set arr (alloc int 5))
    (store (ptradd arr 0) 1)
    (store (ptradd arr 1) 2)
    (store (ptradd arr 2) 3)
    (declare arr2 (ptr int))
    (set arr2 (alloc int 5))))
